jenkins/Jenkinsfile

```diff
@@ -336,15 +334,17 @@ EOT
     }
     stage('Terminate py27 instnace'){ // we made github to wait on a job with this exact name, typo included
```

Comment:
Shouldn't we change the py27 to py36?

Reply:
yes indeed.
Change incoming in a minute

Reply:
FYI, in such cases we also need to update the branch protection to match the stage name.